.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve Juju
+labels:
+  - kind/bug
```

Comment:
Not sure we want to assume it's a bug, until triage or pre-triage.

Reply:
pre-triage can fix that, most bugs reported by this form are likely to be bugs.